themes-default/slim/views/layouts/main.mako

```diff
@@ -97,10 +97,6 @@
         <script type="text/javascript" src="js/config/index.js?${sbPID}"></script>
         <script type="text/javascript" src="js/config/init.js?${sbPID}"></script>
         <script type="text/javascript" src="js/config/notifications.js?${sbPID}"></script>
-        <!-- <script type="text/javascript" src="js/config/post-processing.js?${sbPID}"></script> -->
-        <script src="http://cdn.date-fns.org/v1.0.0/date_fns.js"></script>
```

Comment:
Don't think you meant to do that, but the library is already included in develop,
so next merge of develop into this branch will fix it.